hub/core/dataset/dataset.py

```diff
@@ -2606,9 +2620,10 @@ def copy(
             token=token,
             overwrite=overwrite,
             public=public,
+            unlink=[t for t in self.tensors if self.tensors[t].base_htype != "video"]
```

Comment:
maybe add a constant that allows unlinking of videos too? There could be certain use cases in which user wants an actual materialized dataset including videos as well.